index.js

```diff
@@ -1 +1,41 @@
-console.log("Code your solution!")
+// Selectors
+
+// Event Listeners
+const parent = document.querySelector("form");
+parent.addEventListener("submit", (event) => {
+  event.preventDefault();
+  console.log(userInput);
+
+  const todoList = document.querySelector("ul");
+  const list = document.createElement("li");
+  list.textContent = event.target.userInput.value;
```

Comment:
take a look at the lesson notes to see exactly how you access the value the user inputted.

Reply:
You can ignore this comment